Bezier.cpp

```diff
@@ -4,9 +4,70 @@
 Bezier::Bezier(const Vector2& p0, const Vector2& p1, const Vector2& p2, const Vector2& p3)
 	: p0(p0), p1(p1), p2(p2), p3(p3)
 {
+	// Assign points to Bezier
+	this->p0 = p0;
+	this->p1 = p1;
+	this->p2 = p2;
+	this->p3 = p3;
```

Comment:
This isn't necessary -- these are already initialised in the initialiser list for the constructor (on line 5).